wiki/Tournaments/o!mLN/4/en.md

```diff
@@ -0,0 +1,387 @@
+4. Each player receives the right to use a 2-minute timeout once per match. If players find the need to use the timeout between picks, they should raise their request to the referee facilitating their match.
+5. Players are encouraged to use their timeout only when an urgent situation or personal condition calls for it, and not to stall for time.
+6. In the event where both players are 1 point away from winning the match, the tiebreaker will be played, and the player who wins the tiebreaker map will win the match.
+7. Should there be a tie in map score between the 2 players, the result will be nullified and the map will be played once more to determine who gets the point.
+
+### Additional rules/information
+
+#### Mod configurations
+
+1. The score system used for all matches in this tournament will be **Score V2**.
+2. All maps are played with FreeMod. Visual mods such as Mirror, Hidden, Fade-In, Flashlight, and any combination of these mods are allowed. You are also allowed, but not forced, to play with NoFail.
+
+#### Punctuality
+
+1. Players are required to arrive for their match before the stipulated time for their match. Otherwise, a **10 minute** grace timer will be applied by the appointed referee for the match.
+2. If the player fails to arrive at their match before this timer runs out, the match will be considered as a win by default to the other player.
+3. If the player arrives at their match during this grace period, the time remaining on the timer will be brought forward to the ban/pick proceeding of the match.
+
+#### Ban/Pick procedures
+
+1. Players will be given 2 minutes to roll and perform their first ban/picks. Subsequently, players will have 2 minutes to pick a map and get ready to play the map.
+   - If a player fails to adhere to the timer to ban/pick, the referee will choose a map by random to be picked in place of the player, using the !roll command.
+   - On the second and subsequent occurrences, a point will be automatically awarded to the other player.
+
+#### Disconnection
+
+1. If a player is disconnected from a map within the first 30 seconds of gameplay, they can ask for the map to be aborted and replayed.
+   - This is only available once per match for each player. Subsequent disconnects will give a point to the opponent and use up the remainder of the 10 minute grace period.
+2. In the event a player is unable to return to the lobby before the grace period ends, the match will be considered a win to the other player.
+3. In the event both players are disconnected, the player whose grace period runs out first will concede the match to the opponent.
+4. In the event both players run out of grace period simultaneously, the winner will be the player with the higher qualifier seed.
+
+#### General conduct
+
+1. Every staff and player must respect each other and follow the osu! community rules, no discussion about sensitive topics (for example religion, politics), and anything that could be offensive to others.
```

Comment:
I'd make the second part of this sentence (the "no sensitive topic discussions" clause) a whole separate point. These are two fairly different things (following rules versus no discussions).
```suggestion
1. Every staff and player must respect each other and follow the [osu! community rules](/wiki/Rules).
2. Discussions about sensitive (religion, politics) and other potentially offensive topics are disallowed.
```

Will need to re-index the bullet points bellow.